Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml

```diff
@@ -76,3 +76,132 @@
       department: Justice
       time: 3600 # 1 hours
   - type: GhostTakeoverAvailable
+
+- type: entity
+  name: Silvia
+  id: MobCobraSilvia
+  parent: MobSpaceBasic
+  description: With needle-sharp fangs and an alluring gaze, will you let her bite you?
+  components:
+    - type: HTN
+      rootTask:
+        task: IdleCompound
+    - type: Sprite
+      drawdepth: Mobs
+      sprite: DeltaV/Mobs/Pets/silvia.rsi
+      layers:
+        - map: [ "enum.DamageStateVisualLayers.Base" ]
+          state: silvia
+        - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+          state: glow
+          shader: unshaded
+    - type: FootstepModifier
+      footstepSoundCollection:
+        collection: FootstepSnake
+    - type: MobThresholds
+      thresholds:
+        0: Alive
+        125: Dead
+    - type: DamageStateVisuals
+      states:
+        Alive:
+          Base: silvia
+          BaseUnshaded: glow
+        Dead:
+          Base: dead_silvia
+    - type: Butcherable
+      spawned:
+        - id: FoodMeatSnake
+          amount: 2
+        - id: OmnizineChemistryBottle
+          amount: 1
```

Comment:
should be a special omni-rich meat not you butcher snake and get a glass bottle out of thin air

Reply:
Added a meat